docs/reference/asciidoc-document.md

```diff
@@ -42,6 +42,7 @@ terraform-docs asciidoc document [PATH] [flags]
       --output-values-from string   inject output values from file into outputs (default "")
       --read-comments               use comments as description when description is empty (default true)
       --recursive                   update submodules recursively (default false)
+      --recursive-include-main      include the main module (default true) (default true)
```

Comment:
Looks like you don't need to include `(default true)` in the regular description since then it gets added twice.